Content.Server/Singularity/Components/SinguloFoodComponent.cs

```diff
@@ -6,8 +6,14 @@ namespace Content.Server.Singularity.Components
     [RegisterComponent]
     public sealed partial class SinguloFoodComponent : Component
     {
-        [ViewVariables(VVAccess.ReadWrite)]
         [DataField("energy")]
-        public float Energy { get; set; } = 1f;
+        public float Energy = 1f;
+
+        /// <summary>
+        /// The percentage of the singularity's current energy that this food will drain.
+        /// Only applies when Energy is negative.
+        /// </summary>
+        [DataField("percentageDrain")]
+        public float PercentageDrain = -0.03f;
```

Comment:
```suggestion
        public float Multiplier = 1.0f;
```
since the number is negative I would not call it a drain anymore
Also it is not a percentage.
Just make it a multiplier with default 1. Otherwise the singulo beacon or other entities using this components will also get the drain.